### PR TITLE
1166 detail short url

### LINKED ISSUE #1166: Allow sharing of a short URL from the detail tab
Request from Michael Aye: Have a "share" button on the Detail tab that creates a short URL of the form:

https://opus.pds-rings.seti.org/opus/#/view=detail&detail=co-iss-n1686940931

This would be convenient for posting to Twitter, etc.

### PR DESCRIPTION
**Pull request by dstopp**
- Fixes #1166
- Were any Django, import pipeline, table_schema, or dictionary files modified? N
  - If YES:
    - Database used: opus3_test_210526
    - All Django tests pass: NA
    - FLAKE8 run on modified code: NA
- Were any JavaScript or CSS files modified? Y
  - If YES:
    - JSHINT run on all affected files: Y/NA
    - Tested on Chrome, Firefox, Safari, Edge, and iOS: Y/NA
      (Remember to test all browser sizes)
    - Tested for race conditions (with delayed API calls if applicable): Y/NA

Description of changes:

Known problems:

### CODE REVIEW

**Review by juzen2003 — CHANGES_REQUESTED** [review]
Issues:
- The perfect scrollbar on the detail page is not scrolling, it will show up at the beginning but will be gone once we start to scroll.
- When share button is clicked, the URL will change to the short one and if we refresh the page, the URL will go back to the long one (but we still stay in the detailed tab), not quite sure if this is what we expect.
- Similar to previous one, this exists in previous pull, not related to the changes here. When the cart button is clicked in detailed page, URL will change and if we refresh the page, the page will reload to the search tab.

**Comment by rfrenchseti** [comment]
Both of those last two are definitely bugs. I thought we had found and fixed that second one in the production version but I guess not. Please go ahead and fix it as well.

File: opus/application/static_media/js/detail.js [hunk 1]
```diff
@@ -37,8 +37,16 @@ var o_detail = {
                 case "downloadURL":
                     $(this).attr("href", `/opus/__api/download/${opusId}.zip?urlonly=1&cols=${opus.prefs.cols.join()}`);
                     break;
+                case "share":
+                    if (opusId) {
+                        let urlToShare = $(this).attr("href");
+                        o_detail.copyToClipboard(urlToShare)
+                            .then(() => o_detail.showShareMessage("URL copied!"))
+                            .catch(() => o_detail.showShareMessage("Error copying URL"))
```
Comment (juzen2003) [inline opus/application/static_media/js/detail.js]: miss `;` at the end

**Review by juzen2003 — CHANGES_REQUESTED** [review]
- Database used: opus3_test_210526
- All Django tests pass: Y
- JSHINT run on all affected files: Y
- FLAKE8 run on all affected files: Y
- Tested on Chrome, Firefox, Safari, and iOS: Y
  (Remember to test all browser sizes)
- Tested for race conditions (with delayed API calls if applicable): Y

Only one minor cleanup needed, please add ";" at the end of line 45 in detail.js

**Review by juzen2003 — APPROVED** [review]

**Pull request merged**